addons/common/functions/fnc_loadSettingsLocalizedText.sqf

```diff
@@ -42,7 +41,9 @@ _parseConfigForDisplayNames = {
 
     if !([configFile >> "ACE_Settings" >> _name] call _parseConfigForDisplayNames) then {
         if !([configFile >> "ACE_ServerSettings" >> _name] call _parseConfigForDisplayNames) then {
-            [missionConfigFile >> "ACE_Settings" >> _name] call _parseConfigForDisplayNames;
+            if !([missionConfigFile >> "ACE_Settings" >> _name] call _parseConfigForDisplayNames) then {
+                diag_log text format ["[ACE] - Setting found, but couldn't localize [%1] (server has but we don't?)", _name];
```

Comment:
`text` is uncessary here. `diag_log format["content"]` works without it. Is there a reason is should be here?